pyccel/codegen/printing/ccudacode.py

```diff
@@ -344,15 +344,15 @@ def _print_Allocate(self, expr):
         return '{}\n{}\n{}\n'.format(free_code, shape_Assign, alloc_code)
 
     def _print_Deallocate(self, expr):
```

Comment:
Missing docstring in this function.

Reply:
We don't usually insist on docstrings in `_visit_X` and `_print_X` functions as they would all contain the same thing with only the name of the type changed

Reply:
Okay, I got it.